platforms-serverless/vercel-with-nextjs/prisma/schema-with-napi.prisma

```diff
@@ -0,0 +1,16 @@
+generator client {
+  provider = "prisma-client-js"
+  previewFeatures = ["nApi"]
```

Comment:
https://vercel.com/prisma/e2e-vercel-with-nextjs/933UQzXKzDeUksF75HZknGbxswzS/source?f=src%2Fprisma%2Fschema.prisma shows that this schema is indeed deployed.

But the Client still gets the Query Engine binary, and not the library: 

```
{"files":["index.js","package.json","query-engine-rhel-openssl-1.0.x","schema.prisma"] ...}
```
https://e2e-vercel-with-nextjs-by6g8z242-prisma.vercel.app/api/files

Reply:
Maaayyyyybe there is some special Vercel logic which engine to deploy that does not consider Napi properly? Or something else goes wrong during the build process? 
(Or do we actually upload the local build, which is not using `rhel` but `debian` and then something goes wrong?)

So many possibilities.